plugins/localization-connector/__tests__/memsourceService.spec.ts

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { MemsourceService } from '../src/services/memsourceService'
```

Comment:
this might be dead code

Reply:
Yeah, about the tests, I got fed up because we were changing requirements and they were really hurting my unit tests. I'll create a batch of tests that go in line with the current implementation